### ISSUE TITLE
iGEMM v4r5 contains a divide by zero hazard

### ISSUE THREAD

**Issue by daniellowell** [ROCmSoftwarePlatform/MIOpen#546]
Found in testing PR #147 and blocking that PR.
http://micimaster.amd.com/blue/organizations/jenkins/MLLibs%2FMIOpen/detail/reshuffleAndReduceTestCases/118/pipeline/

```
In file included from gridwise_convolution_forward_implicit_gemm_v4r5_xdlops_nchw_kcyx_nkhw.cpp:2:

In file included from ./gridwise_convolution_forward_implicit_gemm_v4r5_xdlops_nchw_kcyx_nkhw.hpp:5:

In file included from ./tensor_descriptor.hpp:6:

./multi_index_transform.hpp:210:45: error: division by zero is undefined [-Werror,-Wdivision-by-zero]

            idx_low(idim)            = itmp / stride;

                                            ^ ~~~~~~
```

**Comment by zjing14** [comment]
Here is the failed config: ./bin/MIOpenDriver convbfp16 -c 16 -n 1 -k 32 -H 28 -W 28 -u 2 -v 2 -p 3 -q 3 -y 3 -x 3 -F 1

**Comment by aserio** [comment]
@zjing14, can this issue be closed as #547 is now merged?

**Comment by asroy** [comment]
Closing, since #547 is merged
